internal/session/sessionwantsender.go

```diff
@@ -30,12 +30,6 @@ const (
 	BPHave
 )
 
-// SessionWantsCanceller provides a method to cancel wants
-type SessionWantsCanceller interface {
```

Comment:
We no longer cancel wants with the SessionManager - instead the sessions tell the PeerManager when they're no longer interested in a want and it decides whether to cancel the want